tests/functional/api/groups/update_test.py

```diff
@@ -2,17 +2,37 @@
 
 import pytest
 
+from h.models import GroupMembership, GroupMembershipRoles
```

Comment:
TODO: Make the assertions throughout this file more specific. I suspect that the changes in this PR may have actually broken some of the tests here but the tests are still passing because their assertions aren't specific enough.